### PR TITLE
Auto-Correct Private DNS Names Networking Issue

### PR DESCRIPTION
**Pull request by forrestfwilliams**
This is a first attempt at a solution for the Private DNS names error that affects HyP3 ISCE2 burst jobs. The strategy is:

- Add a cron lambda to HyP3 that checks the status of the offending parameter
- Inside the lambda:
    1. If the parameter is set to `True`, change it to `False`
    2. If it is already `False`, do nothing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.4.0]
```
Comment (jtherrmann) [inline CHANGELOG.md:7]:
```suggestion
## [4.3.2]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.3.2]
+### Added
+- Lambda function that checks to ensure that `Enable Private DNS Names` is enabled for endpoints
```
Comment (jtherrmann) [inline CHANGELOG.md:9]:
```suggestion
- Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.3.2]
+### Added
+- Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```
Comment (jtherrmann) [inline CHANGELOG.md:9]:
```suggestion
### Fixed
- Added a Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```

File: apps/disable-private-dns/src/main.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    print('## ENVIRONMENT')
+    vpc_id = os.environ['VPCID']
+    endpoint_name = os.environ['ENDPOINT_NAME']
+    print(f'VPC ID {vpc_id}')
+    print(f'Endpoint Name: {endpoint_name}')
+    print('## EVENT')
+    print(event)
+    print('## PROCESS BEGIN...')
+    disable_private_dns(endpoint_name)
+    print('## PROCESS COMPLETE!')
```
Comment (jtherrmann) [inline apps/disable-private-dns/src/main.py:52]:
```suggestion
    vpc_id = os.environ['VPCID']
    endpoint_name = os.environ['ENDPOINT_NAME']
    print(f'VPC ID {vpc_id}')
    print(f'Endpoint Name: {endpoint_name}')
    disable_private_dns(endpoint_name)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.3.2]
+### Fixed
+- Added a Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```
Comment (jtherrmann) [inline CHANGELOG.md:9]:
```suggestion
- Added a Lambda function that sets `Private DNS names enabled` to false for VPC endpoint.
```

**Review by jtherrmann — APPROVED** [review]
LGTM. I wasn't able to test the Lambda code locally because I think it needs to run from within the VPC itself. Easiest way to test it might be to just merge to UAT.

**Comment by forrestfwilliams** [comment]
Thanks @jtherrmann, merging so that we can take a look at it in UAT

**Pull request merged**